operator/api/v1alpha2/keptnapp_conversion.go

```diff
@@ -0,0 +1,6 @@
+package v1alpha2
+
+// Hub is the stub function to make the API conversion pattern with hub and spokes complete
+func (*KeptnApp) Hub() {
```

Comment:
this would make only `KeptnApp` convertible between `v1alpha1` and `v1alpha2`, right? 
Or is it not necessary in the other structs (e.g., AppVersion) because no field changed and thus the conversion works out of the box?

Reply:
Conversion webhooks (and the corresponding conversion interfaces) are only needed if you actually need to convert some data.
If they're not there, k8s will just convert the `apiVersion` field automatically and leave the rest as it is.